content(cms): create Resource "using-digital-archives-for-social-sciences-research" (#25)
stefanprobst/campus: content/resources/using-digital-archives-for-social-sciences-research.mdx

---
title: Using Digital Archives for Social Sciences Research
lang: en
date: 2021-06-09T00:00:00.000Z
version: 1.0.0
authors:
  - digital-repository-of-ireland
contributors:
  - ocarroll-aileen
  - landy-david
  - kiely-elizabeth
  - gray-jane
  - ryan-maria
tags:
  - e-heritage
  - open-access
  - digital-archives
categories:
  - dariah
abstract: This video recording is of 'Using Digital Archives for Social Sciences
  Research', the third and final webinar in a three-part public lecture series
  hosted by the Digital Repository of Ireland (DRI), aimed at early career
  researchers. The webinar showcases the rich research resources contained in
  digital archival collections that can be used to advance social sciences
  research.
domain: Social Sciences and Humanities
targetGroup: Domain researchers

> type: training-module

Reviewer: video or webinar-recording?

Reviewer: ```suggestion
type: webinar-recording
```

licence: ccby-4.0
remote:
  date: 2021-04-20T00:00:00.000Z
  publisher: Digital Repository of Ireland
  url: https://doi.org/10.7486/DRI.bg25n363c
toc: false
uuid: djD_8l7fIe1MQzvFkDlM3
---

> This screencast is of ‘Using Digital Archives for Social Sciences Research’, the **third** and final webinar in a three-part public lecture series on using digital archives for academic research hosted by the Digital Repository of Ireland (DRI) and aimed primarily at early career researchers.

Reviewer: ```suggestion
This screencast is of "Using Digital Archives for Social Sciences Research", the **third** and final webinar in a three-part public lecture series on using digital archives for academic research hosted by the Digital Repository of Ireland (DRI) and aimed primarily at early career researchers.
```

Access to brick-and-mortar archives has been limited due to COVID-19 restrictions, creating challenges for researchers that rely on archival materials for their research. This webinar aims to showcase some of the rich research resources contained in digital archival collections.

Featured digital collections in this webinar include:

> - ‘Repeal the Eighth and Reproductive Rights’ collection: This collection features 25 interviews conducted with Repeal activists from the project 'Sharing Best Practices in how Civil Society Organisations use the Internet in Organising and Building for Socio-Economic Rights and Trust'. This project looked at how organisations build trust and resolve conflicts in the age of social media. It also explored how digital tools were used to organise and build up the Repeal campaign, develop coalitions and connections, and also how the campaign dealt with the inevitable conflict that all movements have to handle.

Reviewer: ```suggestion
- "Repeal the Eighth and Reproductive Rights" collection: This collection features 25 interviews conducted with Repeal activists from the project "Sharing Best Practices in how Civil Society Organisations use the Internet in Organising and Building for Socio-Economic Rights and Trust". This project looked at how organisations build trust and resolve conflicts in the age of social media. It also explored how digital tools were used to organise and build up the Repeal campaign, develop coalitions and connections, and also how the campaign dealt with the inevitable conflict that all movements have to handle.
```

- National Library of Ireland (NLI) web archival collections: Two NLI collections are featured in the webinar talks. The NLI's General Election 2020 collection encompasses media sites, commentary, and news websites and covers the election campaign, the results, and government formation. At the time of recording, NLI was also undertaking ongoing collection work of the Irish experience of Covid-19. This collection had over 180 websites archived at the time of recording and collecting was expected to continue.

> - Irish Women at Work Oral History Project: This is a collection of 42 oral history recordings, related transcripts, and artefacts collected as part of a feminist research project focused on women’s experiences of work and employment between the 1930s and 1960s. The interviews take a life course approach, focusing on women’s early family lives, education, entry into and experience of the workplace, marriage and motherhood. The narratives provide rich historical insights into diverse jobs, workplace practices, and working conditions.

Reviewer: ```suggestion
- Irish Women at Work Oral History Project: This is a collection of 42 oral history recordings, related transcripts, and artefacts collected as part of a feminist research project focused on women's experiences of work and employment between the 1930s and 1960s. The interviews take a life course approach, focusing on women's early family lives, education, entry into and experience of the workplace, marriage and motherhood. The narratives provide rich historical insights into diverse jobs, workplace practices, and working conditions.
```

> - Life Histories and Social Change: This is a large collection of qualitative life story interviews with three cohorts of Irish citizens, each of which reached adulthood in the crucial decades of the 1950s (an era of socio-economic decline), the 1970s (an era of initial 'modernisation') and in the 1990s (the 'Celtic Tiger' boom). The research was funded by the Irish Research Council and a total of 113 life history interviews were conducted by researchers from Maynooth University between 2006 and 2008.

Reviewer: ```suggestion
- Life Histories and Social Change: This is a large collection of qualitative life story interviews with three cohorts of Irish citizens, each of which reached adulthood in the crucial decades of the 1950s (an era of socio-economic decline), the 1970s (an era of initial 'modernisation') and in the 1990s (the 'Celtic Tiger' boom). The research was funded by the Irish Research Council and a total of 113 life history interviews were conducted by researchers from Maynooth University between 2006 and 2008.
```

- RESCuE-Ireland collection: This collection was created as part of a cross-national European project that aimed to identify the contexts and practices associated with household resilience to the financial crisis of 2008. The dataset consists of 25 narrative interviews with people living in urban and rural settings in an Irish Midlands district.

## Learning outcomes

After watching this webinar, early career researchers will be familiar with some of the digital archival collections that can be explored to advance social sciences research, they will have been introduced to many helpful online research resources, and offered a starting point from which to develop research methodologies for conducting digital archival research.

> <CTA

Reviewer: ```suggestion
<ExternalResource
```
